### PR TITLE
ENDOC-466 Azure install 7.0 update, first draft

### PR DESCRIPTION
**Pull request by jyunmitch**
In the troubleshooting section at the bottom, there is a list of pods that looks wrong. I've listed the new ones below so you can check that these are the correct ones to use, but the status for each pod may be wrong for the error they are talking about.

### CODE REVIEW

File: vuepress/docs/next/tutorials/getting-started/azure-install.md [hunk 1]
```diff
@@ -5,23 +5,21 @@ sidebarDepth: 2
...
 
-- Create a single cluster with up to 5 nodes
-- Install NGINX as an ingress controller
-- Install Entando
+## Prerequisites
 
-If you're already comfortable setting up an AKS cluster and installing NGINX, then you may be able to skip to [setting up Entando](#install-the-entando-custom-resource-definitions).
+- Azure account
+   - **Note:** If you're using an Azure free account, you may need to upgrade your account to enable pay-as-you-go billing. The Azure free account default quota allows just 1-4 vCPU which is not sufficient for this tutorial. There may be a delay before the quotas are updated when you upgrade your account.
+- If you choose not to use the Azure Cloud Shell, you'll also need the Azure command line tool
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/azure-install.md:22]: period at end of sentence

File: vuepress/docs/next/tutorials/getting-started/azure-install.md [hunk 2]
```diff
@@ -32,141 +30,130 @@ If you're already comfortable setting up an AKS cluster and installing NGINX, th
...
-
-Note: A different storage class can be configured by following [these instructions](./gke-install.md#appendix-a-configuring-clustered-storage).
+13. For `Node Pools`, keep the default values
+    - If you're familiar with AKS, you can change these settings as desired
+    - Click `Next: Access`
+14. For `Access`, keep the default values, e.g. System-assigned managed identity
+15. Click `Next: Networking`
+16. Keep the default settings
+17. Click `Review + Create`
+    - Note: There are many other configuration options available for an AKS cluster. Generally, you can change these based on your experience and comfort level with the platform. Entando uses base Kubernetes APIs so as long as you follow the Entando configuration instructions below you can tune your cluster infrastructure to meet your goals.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/azure-install.md:49]: added commas >> Entando uses base Kubernetes APIs, so as long as you follow the Entando configuration instructions below, you can tune...

File: vuepress/docs/next/tutorials/getting-started/azure-install.md [hunk 3]
```diff
@@ -32,141 +30,130 @@ If you're already comfortable setting up an AKS cluster and installing NGINX, th
...
-```
-3. Edit `sample-configmaps/entando-operator-config.yaml` and add the following settings in the data section
-``` yaml
-  entando.requires.filesystem.group.override: "true"
-  entando.ingress.class: "nginx"
-```  
-4. Now create the ConfigMap for the operator
-``` sh
-kubectl apply -f sample-configmaps/entando-operator-config.yaml -n entando
+2. Edit `entando-app.yaml`. Replace `YOUR-HOST-NAME` with `EXTERNAL-IP + .nip.io`. See [this page](../../docs/consume/custom-resources.md#entandoapp) for details on other `EntandoApp` options.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/azure-install.md:127]: See [this page] >> See the [custom resources page]

File: vuepress/docs/next/tutorials/getting-started/azure-install.md [hunk 4]
```diff
@@ -32,141 +30,130 @@ If you're already comfortable setting up an AKS cluster and installing NGINX, th
...
-quickstart-deployer-jj4njqk4bg                       1/1     Running     0          10m
-quickstart-eci-deployer-t0xktqsonk                   0/1     Completed   0          11m
-quickstart-eci-k8s-svc-deployment-78f64c8d89-7c578   1/1     Running     0          11m
-quickstart-kc-deployer-16gzv3clsj                    0/1     Completed   0          13m
-quickstart-kc-server-deployment-7c9bc65744-g52nx     1/1     Running     0          13m
-quickstart-server-deployment-55fcfc6b68-szvkl        0/3     Pending     0          10m
-```
-
-Double check your availability zones. By default an Azure cluster will include nodes from multiple zones, but Azure may not automatically provision their storage.
+If you get an error like: `0/5 nodes are available: 5 node(s) had volume node affinity conflict` or if several deployments fail to start, then you should check your availability zones. By default an Azure cluster will include nodes from multiple zones, but Azure may not automatically provision their storage.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/azure-install.md:152]: By default an Azure cluster>> By default, an Azure cluster

File: vuepress/docs/next/tutorials/getting-started/azure-install.md [hunk 5]
```diff
@@ -5,23 +5,21 @@ sidebarDepth: 2
...
 # Installation on Azure Kubernetes Service (AKS)
 
-## Prerequisites
-
-- Azure account
-   - Note: If you're using an Azure free account, you may need to upgrade your account first to enable pay-as-you-go billing. The Azure free account default quota includes just 1-4 vCPU which is not sufficient for this tutorial. There may be a delay before the quotas are updated when you upgrade your account.
-- If you're not using Azure Cloud Shell, you'll also need:
-  - Azure command line tool
-  - Helm3 client
+The steps below walk you through installing the Entando Platform in an Azure Kubernetes Services (AKS) cluster. This is the process.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/getting-started/azure-install.md:8]: is "This is the process" needed?

File: vuepress/docs/next/tutorials/getting-started/azure-install.md [hunk 6]
```diff
@@ -5,23 +5,21 @@ sidebarDepth: 2
...
+If you're already comfortable setting up an AKS cluster and installing NGINX, then you may be able to skip to [setting up Entando](#install-the-entando-custom-resources).
 
-- Create a single cluster with up to 5 nodes
-- Install NGINX as an ingress controller
-- Install Entando
+## Prerequisites
 
-If you're already comfortable setting up an AKS cluster and installing NGINX, then you may be able to skip to [setting up Entando](#install-the-entando-custom-resource-definitions).
+- Azure account
+   - **Note:** If you're using an Azure free account, you may need to upgrade your account to enable pay-as-you-go billing. The Azure free account default quota allows just 1-4 vCPU which is not sufficient for this tutorial. There may be a delay before the quotas are updated when you upgrade your account.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/getting-started/azure-install.md:21]: **Note:** vs Note: vs > Note vs tip :(

File: vuepress/docs/next/tutorials/getting-started/azure-install.md [hunk 7]
```diff
@@ -32,141 +30,122 @@ If you're already comfortable setting up an AKS cluster and installing NGINX, th
...
-kubectl apply -n entando -f https://raw.githubusercontent.com/entando/entando-releases/v6.3.2/dist/ge-1-1-6/namespace-scoped-deployment/orig/namespace-resources.yaml
+5. Apply the `ConfigMap`
+```sh
+kubectl apply -f entando-operator-config.yaml -n entando
 ```
 
-## Deploy Your Entando Application
-You can now deploy your application to Azure Kubernetes Service.
-1. Download and unpack the entando-helm-quickstart files
+6. Apply the namespace-scoped Custom Resources
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/getting-started/azure-install.md:101]: custom resources ??

File: vuepress/docs/next/tutorials/getting-started/azure-install.md [hunk 8]
```diff
@@ -32,141 +30,122 @@ If you're already comfortable setting up an AKS cluster and installing NGINX, th
...
-1. In your cluster select `Workloads` in the left nav
-2. Click on the deployment for your server application. This is `quickstart-server-deployment` by default
-3. Click on the deployment name inside that application. There will be one
+1. In your cluster, select `Workloads` in the left nav
+2. Click on the deployment for your server application. This is `quickstart-deployment` by default.
+3. Click on the deployment name inside that application. There will be one.
 4. Click on the tab labeled `Conditions`
-5. If you see an error that says `0/5 nodes are available: 5 node(s) had volume node affinity conflict.` Then you need to reconfigure
-your cluster to have nodes in one zone or work with your Azure operations team to provision storage to match node affinity.
+5. If you see an error that says `0/5 nodes are available: 5 node(s) had volume node affinity conflict.` You need to reconfigure your cluster to have nodes in one zone or work with your Azure operations team to provision storage to match node affinity.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/getting-started/azure-install.md:151]: "if" needs a "then"

period outside of message (is currently inside the backtick)

**Review by nshaw — APPROVED** [review]

**Pull request merged**